docs/JAYDEN.md

```diff
@@ -66,81 +113,426 @@
 
 <br /><br /><br />
 
+
 ---
 
+
 # Math: SAT
 
 
 
 
 
+
+# 📐 Triangle Inequality Theorem
+![Triangle Inequality Theorem](./images/Triangle-Inequality-Theorm.webp)
+
+
+## 🎯 What is the Triangle Inequality Theorem?
+
+
+The **Triangle Inequality Theorem** states that:  
+
+
+> **The sum of the lengths of any two sides of a triangle must be *greater* than the length of the third side.**
+
+
+### 🔢 In Math Terms:  
+For any triangle with sides **a**, **b**, and **c**:  
+- ✅ **a + b > c**  
```

Comment:
Too many boldfaced words, boldface must be used a few times to highlight important contexts, not everywhere.